### PR TITLE
up to date with QGIS (checkbox for TMS)

### PR DESCRIPTION
**Pull request by PeterPetrik**

### CODE REVIEW

File: tiles_xyz/tiles_xyz_algorithm.py [hunk 1]
```diff
@@ -455,39 +457,36 @@ def close(self):
...
     def initAlgorithm(self, config=None):
         super(TilesXYZAlgorithmDirectory, self).initAlgorithm()
-        self.conventions = ['Slippy Map', 'TMS']
-        self.addParameter(QgsProcessingParameterEnum(self.TILE_CONVENTION,
-                                                     self.tr('Tile naming convention'),
-                                                     self.conventions,
-                                                     defaultValue=0))
+        self.addParameter(QgsProcessingParameterBoolean(self.TMS_CONVENTION,
+                                                        self.tr('Use inverted tile Y axis (TMS convention)'),
+                                                        defaultValue=False))
```
Comment (wonder-sk) [inline tiles_xyz/tiles_xyz_algorithm.py]: optional=True?

File: tiles_xyz/tiles_xyz_algorithm.py [hunk 2]
```diff
@@ -496,14 +495,14 @@ def processAlgorithm(self, parameters, context, feedback):
...
             html_code = LEAFLET_TEMPLATE.format(
-                tilesetname = "LeafLet Preview",
-                centerx = self.wgs_extent[0] + (self.wgs_extent[2] - self.wgs_extent[0])/2,
-                centery = self.wgs_extent[1] + (self.wgs_extent[3] - self.wgs_extent[1])/2,
-                avgzoom = (self.max_zoom + self.min_zoom) / 2,
-                tilesource = "'file:///{}/{{z}}/{{x}}/{{y}}.{}'".format(output_dir_safe, self.tile_format.lower()),
-                minzoom = self.min_zoom,
-                maxzoom = self.max_zoom,
-                tms = 'true' if is_tms else 'false'
+                tilesetname="LeafLet Preview",
```
Comment (wonder-sk) [inline tiles_xyz/tiles_xyz_algorithm.py]: Leaflet, not LeafLet

**Comment by saberraz** [comment]
Looks good to me.

**Comment by PeterPetrik** [comment]
@saberraz feel free to release

**Pull request merged**